### DISCUSSION TITLE
Adding a mailing list

### DISCUSSION THREAD

**Discussion by jef** [community/community#31532] [Discussions]
I think to totally replace Discourse, it would be awesome to have a mailing list. Perhaps using "watch" and such works with some effort, but it would be nice to have some dedication towards discussions.

**Comment by queenofcorgis** [comment]
Hi @jef - thanks for the feedback. I'd love to know more about what you envision by a mailing list?

**Reply by jef** [reply]
Hey @queenofcorgis! Thanks for getting back to me so quickly 😊

Rereading this now, perhaps mailing list isn't the correct wording 😨 I'm thinking more a "Reader's Digest".

<details><summary>As they say, pictures are worth a thousand words:</summary>

![image](https://user-images.githubusercontent.com/12074633/189968062-d678a73d-6248-4e70-ad19-c30211cbb9b7.png)

</details>

As you know, GitHub sends notifications via Web and/or Email to _everything_ (repositories, discussions, issues, pull requests, etc.) we're subscribed to.

One thing I would envision is having an email (or notification) to a subscribed user only for discussions. Perhaps in the preferences they could select some sort of cadence.

**Reply by queenofcorgis** [reply]
@jef awesome idea 😃 ! We're still continuing to ramp up as we get settled on Discussions so I've passed this feedback along to our team. Appreciate you sharing and expanding - let me know if you have any other ideas on how we can make your Community on Discussions experience a great one.

**Reply by jef** [reply]
Thank you!
